README.md

```diff
@@ -114,23 +115,26 @@ $ ./multinode-demo/validator.sh ubuntu@10.0.1.51:~/solana 10.0.1.51
 To run a performance-enhanced leader or validator (on Linux),
 [CUDA 9.2](https://developer.nvidia.com/cuda-downloads) must be installed on
 your system:
+
 ```bash
 $ ./fetch-perf-libs.sh
 $ SOLANA_CUDA=1 ./multinode-demo/leader.sh
 $ SOLANA_CUDA=1 ./multinode-demo/validator.sh ubuntu@10.0.1.51:~/solana 10.0.1.51
-
 ```
 
 
 
 Testnet Client Demo
 ---
 
-Now that your singlenode or multinode testnet is up and running, in a separate shell, let's send it some transactions! Note we pass in
-the JSON configuration file here, not the genesis ledger.
+Now that your singlenode or multinode testnet is up and running let's send it some transactions! Note that we pass in
+the expected number of nodes in the network.  If running singlenode pass 1, if multinode pass the number
```

Comment:
Recommend punctuation to reduce confusion:
`If running singlenode, pass 1; if multinode, pass the number...`